integration_test/third_party_apps_data/applications/nvml/exercise

```diff
@@ -0,0 +1,91 @@
+# build and run a test application that generates CUDA traffic
```

Comment:
Can we just use one of the demos that ships with the cuda library? It looks like it includes a `cuda_demo_suite` that includes `vectorAdd` and `busGrind` that both look like they'll generate CUDA traffic. (That should make this script much simpler)

Reply:
Good point! Replaced with `bandwidthTest` since this seems to be the only program that can run longer - longer than the default 60s collection interval for the collector to pick the process up.